src/components/elements/TimePicker.js

```diff
@@ -17,7 +18,26 @@ class TimeAndDatePicker extends Component {
 
   constructor () {
     super()
-    this.state = { startDate: null, startTime: null, endDate: null, endTime: null }
+    this.state = {
+      endDate: DateTime.local().toISODate().toString(),
+      endTime: DateTime.local().toISOTime().toString(),
+      startDate: DateTime.local().minus({day: 1}).toISODate().toString(),
```

Comment:
Instead of seperating out the date and time, pass a datetime string…

Reply:
Assign and reuse a `now` variable so each time is relative to the same timepoint. (something like `let now = DateTime.utc()`

Reply:
Sure.